toolkit/tools/imagecustomizerapi/module.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+)
+
+type Modules struct {
+	Load   []Module `yaml:"Load"`
+	Unload []Module `yaml:"Unload"`
+}
+
+type Module struct {
+	Name string `yaml:"Name"`
+}
+
+func (m *Module) IsValid() error {
```

Comment:
nit: You should probably have the order of the `IsValid()` functions match the order of the structs.